features/facebookImport/src/importer/facebook-account.js

```diff
@@ -3,6 +3,7 @@ class FacebookAccount {
         this._importingResults = [];
         this._importedFileNames = [];
 
+        this._name = null;
```

Comment:
Nit: empty string for consistency with the rest of the initializations?

Reply:
Indeed, empty string fits better.